Doc/tools/extensions/pyspecific.py

```diff
@@ -536,6 +536,7 @@ def process_audit_events(app, doctree, fromdocname):
         nodes.colspec(colwidth=30),
         nodes.colspec(colwidth=55),
         nodes.colspec(colwidth=15),
+        cols=3
```

Comment:
nice, I had almost the same patch for deadsnakes (ubuntu packaging) but opted for a trailing comma here: https://github.com/deadsnakes/python3.8/blob/dc6234da93ec2b4be6535209a9b63a4541217c8c/debian/patches/0034-fix-docs-build.patch

Reply:
OK, added a comma.